File: LeetCode/Minimum Number of Arrows to Burst Balloons.cpp
class Solution {
public:
    int findMinArrowShots(vector<vector<int>>& points) {
        sort(points.begin(), points.end(), [](const vector<int> &a, const vector<int> &b){
            return a[1] < b[1];
        });
            
        long long lastArrow = points[0][1];
        long long ans = 1;
            
        for (int i=1; i < points.size(); i++) {
            if (points[i][0] > lastArrow) {
                ans++;
                lastArrow = points[i][1];
            }
        }
        
        return ans;
    }
};

File: LeetCode/Minimum Number of Arrows to Burst Balloons.py
class Solution:
    def findMinArrowShots(self, points: List[List[int]]) -> int:
        points.sort(key=lambda x: x[1])
        
        lastArrow = points[0][1]
        ans = 1
        
        for i in range(1, len(points)):
            if points[i][0] > lastArrow:
                ans+=1
                lastArrow = points[i][1]
    
        return ans
            